source/process/community-guidelines.rst

```diff
@@ -0,0 +1,141 @@
+============================================================
+Mattermost Community Guidelines
+============================================================
+
+In this document we set out some easy-to-follow community guidelines, including:
+
+- `GitHub Help Wanted Issues in Platform Repository`_ - Guidelines for managing Help Wanted issues in GitHub
```

Comment:
Is there any way we could shorten / make the formatting look a little better here? Not a big deal if we can't, the first line is just a little long

<img width="687" alt="screenshot 2016-12-15 14 18 53" src="https://cloud.githubusercontent.com/assets/9354841/21238492/46a94f62-c2d1-11e6-811c-df3cd1bbab71.png">